charts/zalenium/values.yaml

```
@@ -143,15 +143,20 @@ persistence:
     size: 80Gi
 
 ingress:
-  enabled: false
+  enabled: true
   tls: false
   # secretName: my-tls-cert # only needed if tls above is true
   hostname: zalenium.foobar.com
+  # context path without the forward slash (/)
+  # leaving this empty would run zalenium in the root context (/)
+  contextpath: 
```

Then put a default value so it is just easier to enable/disable the ingress: 
```suggestion
  contextpath: /
```

---

Made this change